lib/Twig/SimpleFunction.php

```diff
@@ -87,4 +89,14 @@ public function isVariadic()
     {
         return $this->options['is_variadic'];
     }
+
+    public function isDeprecated()
+    {
+        return $this->options['deprecated'];
+    }
+
+    public function getAlternative()
```

Comment:
I don't think it is worth it. People should first move away from these deprecated classes and then add the deprecation notice.